bigtable/type.go

```diff
@@ -32,6 +36,14 @@ func (u unknown[T]) proto() *T {
 	return u.wrapped
 }
 
+// MarshalJSON returns the string representation of the protobuf.
+func (u unknown[T]) MarshalJSON() ([]byte, error) {
+	if t, ok := any(u.wrapped).(proto.Message); ok {
+		return protojson.MarshalOptions{}.Marshal(t)
```

Comment:
Consider using the same marshal settings we use in GAPICs. Example: https://github.com/googleapis/google-cloud-go/blob/60ad7f32d69ed3986fb1326841bba2c5dc36e508/secretmanager/apiv1/secret_manager_client.go#L950

Reply:
Done